src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -1070,13 +1070,19 @@ public function testDumpFile()
     {
         $filename = $this->workspace.DIRECTORY_SEPARATOR.'foo'.DIRECTORY_SEPARATOR.'baz.txt';
 
+        // skip mode check on Windows
+        if ('\\' !== DIRECTORY_SEPARATOR) {
+            $oldMask = umask(0002);
```

Comment:
If we wanted to improve the tests this way, we should do that on the `2.3` branch imo.